src/test/java/com/fitwsarah/fitwsarah/accountsubdomain/presentationlayer/AccountControllerIntegrationTest.java

```diff
@@ -107,5 +113,27 @@ private String asJsonString(Object obj) throws JsonProcessingException {
         return objectMapper.writeValueAsString(obj);
     }
 
+    public String obtainAuthToken() throws Exception {
```

Comment:
What is this function used for exactly?

Reply:
This obtain the token directly from the auth0 and uses it inside the test so we don't have problem with the unauthorized for the test.